docs/build/sdks/js/zksync-ethers/paymaster-utils.md

```diff
@@ -11,18 +11,11 @@ The [paymaster utilities library](https://github.com/zksync-sdk/zksync-ethers/bl
 
 ## Contract interfaces
 
-### `IPaymasterFlow` Deprecated
-
-::: warning Deprecated
-
-- This method is deprecated in favor of [utils.PAYMASTER_FLOW_ABI](./utils.md#paymasterflow).
-  :::
-
 Constant ABI definition for
-the [Paymaster Flow Interface](https://github.com/matter-labs/era-contracts/blob/87cd8d7b0f8c02e9672c0603a821641a566b5dd8/l2-contracts/contracts/interfaces/IPaymasterFlow.sol).
+the [Paymaster Flow Interface](https://github.com/matter-labs/era-contracts/blob/main/l2-contracts/contracts/interfaces/IPaymasterFlow.sol).
```

Comment:
We should link to hashes for GitHub code, in case those files are migrated or renamed

Reply:
I changed the link to point to appropriate commit.